librz/util/print.c

```diff
@@ -1726,43 +1774,50 @@ RZ_API int rz_print_get_cursor(RzPrint *p) {
 	return p->cur_enabled ? p->cur : 0;
 }
 
-RZ_API int rz_print_jsondump(RzPrint *p, const ut8 *buf, int len, int wordsize) {
+/**
+ * \brief Print dump in json format
+ * \param p RzPrint instance
+ * \param buf Buffer to print to
+ * \param len Print only this many bytes
+ * \param wordsize Size of a word in bits
+ * \return Dump JSON string
+ */
+RZ_API char *rz_print_jsondump_str(RzPrint *p, const ut8 *buf, int len, int wordsize) {
```

Comment:
Looks like `p` here is unused, maybe remove it?

Reply:
![image](https://user-images.githubusercontent.com/24983369/180590592-5f48959a-2f9a-42ee-bb2d-31b9697d777c.png)
maybe not completely unused